modules/UI/videolayout/RemoteVideo.js

```diff
@@ -188,37 +189,66 @@ RemoteVideo.prototype._generatePopupContent = function () {
     const { isModerator } = APP.conference;
     const participantID = this.id;
 
+    const buttons = [];
+
     /* jshint ignore:start */
-    return (
-        <RemoteVideoMenu id = { participantID }>
-            { isModerator
-                ? <MuteButton
-                    isAudioMuted = { this.isAudioMuted }
-                    onClick = { this._muteHandler }
-                    participantID = { participantID } />
-                : null }
-            { isModerator
-                ? <KickButton
-                    onClick = { this._kickHandler }
-                    participantID = { participantID } />
-                : null }
-            { remoteControlState
-                ? <RemoteControlButton
-                    onClick = { onRemoteControlToggle }
-                    participantID = { participantID }
-                    remoteControlState = { remoteControlState } />
-                : null }
-            { onVolumeChange
-                ? <VolumeSlider
-                    initialValue = { initialVolumeValue }
-                    onChange = { onVolumeChange } />
-                : null }
-        </RemoteVideoMenu>
-    );
+    if (isModerator) {
```

Comment:
I had to shove all this logic into a new component to get InlineDialog working. It wasn't ideal but I think your changes can be copied over 1 for 1. (I'm thinking one ideal change for the future would be to pass the remote video menu as a child prop  but I'm not set on that right now.)